integration-libs/cds/src/profiletag/services/profiletag-event.service.ts

```diff
@@ -104,7 +104,17 @@ export class ProfileTagEventService {
     this.exposeConfig(newConfig);
   }
 
+  /*
+   * Checks if the script with the given source exists in the document or not.
+   */
+  private isScriptLoaded(scriptSource: string): boolean {
+    return document.querySelectorAll('[src="' + scriptSource + '"]').length > 0;
+  }
+
   private addScript(): void {
+    if (this.isScriptLoaded(this.config.cds.profileTag.javascriptUrl)) {
```

Comment:
Please add a unit test to verify that the script still has 1 instance of the script element when loading twice

Reply:
Also addressed.